business_register/pep_scoring/rules.py

```diff
@@ -104,3 +104,75 @@ def calculate_weight(self):
             }
             return weight, data
         return 0, {}
+
+
+class IsLiveNowhereCity(BaseScoringRule):
+    """
+    Rule 4.1 - PEP04_adr
+    weight - 0.7
+    There is no information on the real estate or apartment in the city, which indicated as PEP's place of residence
+    """
+
+    def calculate_weight(self):
+        property_types = [Property.HOUSE, Property.SUMMER_HOUSE, Property.APARTMENT, Property.ROOM]
+        declarations_id = Declaration.objects.filter(
+            pep_id=self.pep.id,
+        ).values_list('id', flat=True)[::1]
+        for declaration in declarations_id:
+            city = Declaration.objects.filter(
+                id=declaration,
+            ).values_list('city_of_residence_id', flat=True)[::1][0]
+            property_cities = Property.objects.filter(
+                declaration=declaration,
+                type__in=property_types,
+            ).values_list('city_id', flat=True)[::1]
+            if city not in property_cities:
+                weight = 0.7
+                data = {
+                    "declaration_id": declaration,
+                    "live_in_city": city,
+                }
```

Comment:
data should be `json` serializable... for example
```suggestion
                data = {
                    "declaration_id": declaration,
                    "live_in_city": city.name,
                    "live_in_city_id": city.id,
                }
```

Reply:
this also applies to other rules